gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/HadoopFileSystemIntegrationHelper.java

```diff
@@ -243,7 +243,7 @@ protected String readTextFile(Path hadoopPath, int offset, int len, boolean chec
           .that(bytesReadStats)
           .isEqualTo(len);
     } else if (statistics == FileSystemStatistics.GREATER_OR_EQUAL) {
-      assertWithMessage("Expected %d <= %d", len, bytesReadStats)
+      assertWithMessage("Expected %s <= %s", len, bytesReadStats)
```

Comment:
why we need this change?

Reply:
Updated Error Prone caught the bug - `assertWithMessage` does not support any placeholders except `%s`.